README.md

```diff
@@ -107,16 +95,13 @@ controller.setFocusBehavior("focus-captured-surface");
 ```js
 // INCORRECT USAGE #2.
 
-const controller = new CaptureController;
-const stream = await navigator.mediaDevices.getDisplayMedia({
-  video: true,
-  controller: controller
-});
+const controller = new CaptureController();
+const stream = await navigator.mediaDevices.getDisplayMedia({ controller });
 
 // This task will be queued behind the the task
 // that closes the window of opportunity to
 // call setFocusBehavior().
 setTimeout(() => {
   controller.setFocusBehavior("focus-captured-surface");
-});
+}, 0);
```

Comment:
I'm OK with doing this more explicitly, but I wonder - why did you add that, given that your original direction was to be terser? It's an optional parameter defaulting to 0, is it not?

> The time, in milliseconds that the timer should wait before the specified function or code is executed. If this parameter is omitted, a value of 0 is used, meaning execute "immediately", or more accurately, the next event cycle.

Taken from [here](https://developer.mozilla.org/en-US/docs/Web/API/setTimeout).

Reply:
I didn't realize it was optional. 🤦  You're right, let's not add `0`

Reply:
I'm fine either way. I thought it was common knowledge enough to not require being explicit, but if you were unaware, surely many others don't either.

Reply:
Let's not add `0`

Reply:
Okay.